models.py

```diff
@@ -17,8 +18,10 @@
 
 
 
-def connect():
+
+def test_connection():
```

Comment:
you just name this method `connect` seeing as it is doing the connection for you.